README.md

```diff
@@ -38,24 +38,24 @@ See the full list of [available commands](https://aka.ms/azure-devops-cli-comman
 
 ### Login (setup credentials)
 
```

Comment:
Mention az login.

Reply:
Yeah, az login should be one of the options. Thanks!